Skycat doc2

JoanneBogart opened the pull request:
Installation and usage instructions as well as description of catalogs for ops rehearsal and roman/rubin simulations.

See it [here](https://lsstdesc.org/skyCatalogs/)

matroxel commented:
Looks good to me. Only minor suggestion would be to write out what SSO is on the main page where the SSO file is defined for clarity.

JoanneBogart commented:
@matroxel  I followed your suggestion about SSO.  I also fixed a couple typos and updated the documentation workflow to publish from the main branch rather than my development branch. Please take a look when you have a chance and approve if it seems ok so I can merge. Thanks!

matroxel reviewed (approved):
Thanks!
